src/test/java/fr/cnes/sonar/plugins/icode/model/ModelTest.java

```diff
@@ -40,10 +40,6 @@ public void before() {
         project.setAnalysisInformations(new AnalysisInformations());
         project.getAnalysisInformations().setAuthor("Me");
 
```

Comment:
As suggested by SonarCloud some of these tests can be refactored, maybe it can be a good idea to do it to avoid SonarSource refusing the release.